opensrp-chw-core/src/main/java/org/smartregister/chw/core/utils/CoreConstants.java

```diff
@@ -100,7 +100,7 @@
         public static final String FAMILY_MEMBER_REGISTER = "family_member_register";
         public static final String ANC_REGISTER = "anc_register";
         public static final String MALARIA_REGISTER = "anc_malaria_confirmation";
-        public static final String GLOBAL_CONFIGS = "global_configs";
+        public static final String INDICATOR_CONFIGS = "indicator_configs";
```

Comment:
Is this supposed to a change in this PR? I assume that this would be a breaking change

Reply:
Yes, I updated it after testing the app with these changes. The payload from the server has the identifier as `indicator_configs`

> `"identifier": "indicator_configs",`